plumbing/storer/object.go

```diff
@@ -40,6 +40,8 @@ type EncodedObjectStorer interface {
 	// HasEncodedObject returns ErrObjNotFound if the object doesn't
 	// exist.  If the object does exist, it returns nil.
 	HasEncodedObject(plumbing.Hash) error
+	// Close any opened files.
+	Close() error
```

Comment:
What about having `Close()` only on the filesystem implementation, not on the general interface?

Reply:
Somehow I understood that you didn't want to expore `Close` in `git.Repository`. It's not deleted from `EncodedObjectStorer`.